### DISCUSSION TITLE
How do I go about sanitizing user input?

### DISCUSSION THREAD

**Discussion by prests** [mdx-editor/editor#440] [Q&A]
I've looked into some examples people have used to sanitize user input for [general mdx](https://github.com/hashicorp/next-mdx-remote/blob/5ca106487cae7dfdb96af636d7c316c40f079108/src/serialize.ts#L41), but a lot of the examples output HTML. From my understanding the editor just expects a markdown string and cannot handle a Vfile as an input. Is there a plugin that accomplishes this? How have people been handling this case?

**Comment by petyosi** [comment]
Perhaps I am missing something - the link above seems to be related to the serialization of MDX for the rendering mechanisms of next-mdx-remote. How is that related to sanitization? 

Apart from that, reliable input sanitization should happen on the server, so I don't think there's anything specific to the editor output.
